.circleci/Dockerfile

```diff
@@ -6,13 +6,16 @@ RUN go get -u github.com/google/wire/cmd/wire
 
 RUN go get -u golang.org/x/tools/cmd/goimports
 
-RUN sudo apt-get install default-jdk
```

Comment:
Since there was no `apt-get update` before, this was now failing; I went ahead and combined it with the NodeJS + Yarn installs with cleanup in the process of fixing